config/routes.rb

```diff
@@ -30,6 +54,12 @@
         resources :courses
       end
     end
+    namespace :v2 do
+      resources :providers
+      resources :subjects
+      resources :courses
+      resources :organisations
+    end
```

Comment:
These don't have controllers (yet). That means if someone hits these endpoints they get a 500 not a 404. I think we should only add these routes in when we have a controller to respond to it.

Reply:
I was getting distracted with this...
https://github.com/rails-api/active_model_serializers/issues/144

regards to 404 vs 500

![image](https://user-images.githubusercontent.com/470137/52488728-66c9f880-2bb8-11e9-8411-c5b115b0982d.png)

Reply:
Ah, good spot. I still wonder about adding the routes first without having controllers for them, but at least it doesn't 500 then.